koku/api/provider/models.py

```diff
@@ -481,7 +489,7 @@ def _delete_from_target(self, target_info, target_values=None):
   from {qual_table_name}
  where "{target_info["column_name"]}" = any(%s)
 ;
-"""
+"""  # noqa: E231 E702 E272
```

Comment:
What causes the need for this?

Reply:
just to stop pre-commit complaining instead of running `--no-verify`. I can undo these though if you prefer.